pages/[slug]/dashboard.tsx

```diff
@@ -125,77 +119,132 @@ const MemberList = ({
 const IDPList = ({
   user,
   saml_connections,
-}: Pick<Props, "user" | "saml_connections">) => {
-  const [idpName, setIDPName] = useState("");
-  const router = useRouter();
+  oidc_connections }: Pick<Props, "user" | "saml_connections" | "oidc_connections">) => {
+    const [idpNameSAML, setIdpNameSAML] = useState("");
+    const [idpNameOIDC, setIdpNameOIDC] = useState("");
+    const [isSaml, setIsSaml] = useState(true);
 
-  const onCreate: FormEventHandler = async (e) => {
-    e.preventDefault();
-    const res = await createSamlSSOConn(idpName);
-    const conn = await res.json();
-    await router.push(
-      `/${router.query.slug}/dashboard/saml/${conn.connection_id}`
-    );
-  };
+    const router = useRouter();
+
+    const onSamlCreate: FormEventHandler = async (e) => {
+        e.preventDefault();
+        const res = await createSamlSSOConn(idpNameSAML);
+        if (res.status !== 200) {
+            alert("Error creating connection, are you at the max # of SAML connections (5)?");
```

Comment:
I would remove a set "max" of SAML/OIDC connections as it depends on billing permissions as well

Reply:
✅